src/gsy_e/models/area/__init__.py

```diff
@@ -221,22 +235,25 @@ def area_reconfigure_event(self, **kwargs):
                             export_capacity_kVA=export_capacity_kVA
                         )
 
-        except Exception as ex:
+        except (GSyAreaException, GSyDeviceException) as ex:
```

Comment:
What about these exceptions, won't we have to rename them or did you leave them as a follow up PR?

Reply:
Umm I'm okay with sticking to the requirements but I prefer to also get rid of these terms completely to not create confusions in the future (even for us current developers)
No need to do it in context of this PR, but I vote for doing it as a follow up one

Reply:
Ok, I can work on creating a new PR to fix this.